diff --git a/Users/rhyssullivan/src/executor/node_modules/postgres/.bun-tag-143b5ae176ff48e1 b/.bun-tag-143b5ae176ff48e1
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/Users/rhyssullivan/src/executor/node_modules/postgres/.bun-tag-4eedc358e39b7c2a b/.bun-tag-4eedc358e39b7c2a
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/Users/rhyssullivan/src/executor/node_modules/postgres/.bun-tag-9ff967fa16bcf459 b/.bun-tag-9ff967fa16bcf459
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/Users/rhyssullivan/src/executor/node_modules/postgres/.bun-tag-a742c4db68849f09 b/.bun-tag-a742c4db68849f09
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/cf/polyfills.js b/cf/polyfills.js
index 53c5203dd5b057d741d14d708ba83b27751b527c..9e5db3cde4e2a17c6e45b28621058259c193f1a6 100644
--- a/cf/polyfills.js
+++ b/cf/polyfills.js
@@ -116,7 +116,10 @@ export const tls = {
     tcp.raw = tcp.raw.startTls({ servername })
     tcp.raw.closed.then(
       () => tcp.emit('close'),
-      (e) => tcp.emit('error', e)
+      (e) => {
+        if (e && e.message === 'Stream was cancelled.') return tcp.emit('close')
+        tcp.emit('error', e)
+      }
     )
     tcp.writer = tcp.raw.writable.getWriter()
     tcp.reader = tcp.raw.readable.getReader()
@@ -155,7 +158,10 @@ function Socket() {
             ? close()
             : ((tcp.readyState = 'open'), tcp.emit('secureConnect'))
         },
-        (e) => tcp.emit('error', e)
+        (e) => {
+          if (e && e.message === 'Stream was cancelled.') return close()
+          tcp.emit('error', e)
+        }
       )
       tcp.writer = tcp.raw.writable.getWriter()
       tcp.reader = tcp.raw.readable.getReader()
@@ -179,7 +185,10 @@ function Socket() {
   }
 
   function write(data, cb) {
-    tcp.writer.write(data).then(cb, error)
+    tcp.writer.write(data).then(cb, (err) => {
+      if (err && err.message === 'Stream was cancelled.') return
+      error(err)
+    })
     return true
   }
 
@@ -201,13 +210,19 @@ function Socket() {
       while (({ done, value } = await tcp.reader.read(), !done))
         tcp.emit('data', Buffer.from(value))
     } catch (err) {
-      error(err)
+      if (!(err && err.message === 'Stream was cancelled.'))
+        error(err)
     }
   }
 
   async function readFirst() {
-    const { value } = await tcp.reader.read()
-    tcp.emit('data', Buffer.from(value))
+    try {
+      const { value } = await tcp.reader.read()
+      tcp.emit('data', Buffer.from(value))
+    } catch (err) {
+      if (!(err && err.message === 'Stream was cancelled.'))
+        error(err)
+    }
   }
 
   function error(err) {
